pep-0690.rst

```diff
@@ -131,9 +131,26 @@ dictionaries in general.
 Specification
 =============
 
-Lazy imports are opt-in, and globally enabled either via a new ``-L`` flag to
-the Python interpreter or via a call to a new ``importlib.set_lazy_imports()``
-function, which makes all imports after the call lazy.
+Lazy imports are opt-in, and they can be:
+
+* Globally enabled, either via a new ``-L`` flag to the Python interpreter or
+  via a call to a new ``importlib.set_lazy_imports()`` function, which makes
+  *all* new imports after the call immediately lazy.
+
+* In module enabled, via ``importlib.enable_lazy_imports_in_module()``.
+
+When the flag ``-L`` is passed to the Python interpreter, a new
+``sys.flags.lazy_imports`` is set to ``True``, otherwise it exists as ``False``.
+This flag does not reflect whether the current status of lazy imports is enabled
+or disabled (as a result of using ``importlib.set_lazy_imports()`` or
+``importlib.enable_lazy_imports_in_module()``). The ``-L`` flag is propagated
+from to new Python subprocesses by using ``sys.flags.lazy_imports``, so in an
+analogous way, new subprocesses don't necessarily reflect the current status of
+lazy imports at the time the subprocess was spawned either.
+
+Actual current status of whether lazy imports is enabled or not at any moment
+can be retrieved using ``importlib.is_lazy_imports_enabled()``, which will
+return ``True`` if lazy imports is enabled or ``False`` otherwise.
```

Comment:
the subtleties of `-L`, programmatic enablement, and subprocesses, feel overwhelming at this point of the spec. I suggest discussing the subtleties in a later sub-heading, and keep it simple here